runhouse/resources/serverless/aws_lambda_function.py

```diff
@@ -603,9 +624,9 @@ def to(
         # if function exist - will update it. Else, a new one will be created.
         if self._lambda_exist(self.name):
             # updating the configuration with the initial configuration.
-            # TODO{SB}: enable the user to change the config of the Lambda.
+            # TODO [SB]: enable the user to change the config of the Lambda.
             # lambda_config = self._update_lambda_config(env_vars)
-            lambda_config = LAMBDA_CLIENT.get_function(FunctionName=self.name)
+            lambda_config = self.LAMBDA_CLIENT.get_function(FunctionName=self.name)
 
         else:
             # creating a new Lambda function, since it's not existing in the AWS account which is configured locally.
```

Comment:
similar note as above, we don't need to pass `env_vars` around if we can access them through the env object